language/documentation/examples/experimental/verify-sort/sources/loop.move

```diff
@@ -0,0 +1,66 @@
+// exclude_for: cvc5
+module 0x42::VerifySort {
+
+    use std::vector;
+    // iperm is a ghost variable for verification
+    public fun verify_sort(v: &mut vector<u64>, iperm: &mut vector<u64>) {
+        let vlen = vector::length(v);
+        spec {
+            assume vlen == 45;
```

Comment:
Why is this assumed to be 45?

Reply:
It does not have to be 45. Any number I tested works here.